utils/getInvoices/index.ts

```diff
@@ -378,16 +378,17 @@ export const getCleaningInvoice = ({
 
     return {
       type: invoice.type,
-      price: +toRoundFixed(+invoice.sum || +invoice.price),
+      price: +toRoundFixed(+invoice.price),
       sum: +toRoundFixed(+invoice.sum || +invoice.price),
     }
   }
 
-  if (!isEmpty(company?.cleaning) && !isNaN(company.cleaning)) {
+  if (company?.cleaning || !isNaN(company?.cleaning)) {
+
     return {
       type: ServiceType.Cleaning,
-      price: +toRoundFixed(company.cleaning),
-      sum: +toRoundFixed(company.cleaning),
+      price: +toRoundFixed(company?.cleaning),
+      sum: 0,
```

Comment:
Revert this file to its original state and try to run all tests again. If it doesn't work, write about it in the comments or DM me